packages/components-native/src/Text/Text.tsx

```diff
@@ -70,6 +70,11 @@ interface TextProps
    */
   readonly italic?: boolean;
 
+  /**
+   * Underline text
```

Comment:
if the `dotted` value isn't supported/implemented on Android, at the very least we should call that out in the JSDocs here

I think it's reasonable for a consumer to assume that it works on both platforms

Reply:
Addressed in e9900e023d046db9397ce4e7f083b93e9b47e7b3